src/views/blocks/rsvp/form/form.php

```diff
@@ -11,10 +11,13 @@
  * @link {INSERT_ARTICLE_LINK_HERE}
  *
  * @since 4.9
- * @version 4.9.4
+ * @since TBD Add $post_id to filter for hiding opt-outs.
+ *
+ * @version TBD
  *
  */
 $ticket_id   = $this->get( 'ticket_id' );
+$post_id     = $this->get( 'post_id' );
```

Comment:
:no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $post_id

<sub>hash: aecb3aa24affd099028e936cbce2b935</sub>